docker/pwsh-exchange/Dockerfile

```diff
@@ -3,4 +3,6 @@ FROM demisto/powershell-ubuntu:7.1.1.16151
 
 RUN pwsh -c "Set-PSRepository -Name PSGallery -InstallationPolicy Trusted -ErrorAction Stop"
 RUN pwsh -c "Install-Module -ErrorAction Stop -Scope AllUsers -Name ExchangeOnlineManagement -RequiredVersion 2.0.4-Preview4 -AllowPrerelease -Force"
-RUN pwsh -c "Install-Module -Name SelfSignedCertificate -AllowPrerelease -Force"
\ No newline at end of file
+RUN pwsh -c "Install-Module -Name SelfSignedCertificate -AllowPrerelease -Force"
+RUN pwsh -c "Install-Module MicrosoftTeams -Force"
+RUN pwsh -c "Import-Module MicrosoftTeams -Force"
```

Comment:
You should put this `Import-Module` in the integration itself as the programmer should be aware of the import.